src/main/java/utils/reusable/specifications/ReusableSpecifications.java

```diff
@@ -0,0 +1,47 @@
+package utils.reusable.specifications;
+
+import io.restassured.authentication.PreemptiveOAuth2HeaderScheme;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.hamcrest.Matchers.lessThan;
+import static utils.constants.TestConstants.GO_REST_ACCESS_TOKEN;
+
+public class ReusableSpecifications {
+
+    public static RequestSpecBuilder rspec;
```

Comment:
Please declare it under each method. If a field is static it will be shared between all instantiates of the same object.
I did a test and modified the first name for User to be static. Also note, the getters and setters will not be generated automated so you need to manually create them and return the firstname (not this.firstname, since abstract classes cannot be instantiated).
![image](https://user-images.githubusercontent.com/12393720/66605993-0ce75b80-eba9-11e9-8ed4-1e77e77f61a9.png)